docs/src/content/docs/hi/guides/customization.mdx

```diff
@@ -0,0 +1,448 @@
+
+```js {9-11}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: 'संपादन लिंक वाले दस्तावेज़',
+			editLink: {
+				baseUrl: 'https://github.com/withastro/starlight/edit/main/docs/',
+			},
+		}),
+	],
+});
+```
+
+## कस्टम 404 पेज
+
+Starlight साइटें डिफ़ॉल्ट रूप से एक साधारण 404 पेज प्रदर्शित करती हैं।
+आप अपनी `src/content/docs/` निर्देशिका में `404.md` (या `404.mdx`) फ़ाइल जोड़कर इसे अनुकूलित कर सकते हैं:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - **404.md**
+      - index.md
+- astro.config.mjs
+
+</FileTree>
+
+आप अपने 404 पेज में Starlight के सभी पेज लेआउट और अनुकूलन तकनीकों का उपयोग कर सकते हैं। उदाहरण के लिए, डिफ़ॉल्ट 404 पेज फ्रंटमैटर में [`splash` template](#पेज-लेआउट) और [`hero`](/hi/reference/frontmatter/#hero) घटक का उपयोग करता है:
+
+```md {4,6-8}
+---
+# src/content/docs/404.md
+title: '404'
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: पेज नहीं मिला। URL जांचें या खोज बार का उपयोग करने का प्रयास करें।
+---
+```
+
+### डिफ़ॉल्ट 404 पेज को अक्षम करना
+
+यदि आपके परियोजना को पूरी तरह से अनुकूलित 404 लेआउट की आवश्यकता है, तो आप `src/pages/404.astro` रूट बना सकते हैं और Starlight के डिफ़ॉल्ट रूट को अक्षम करने के लिए [`disable404Route`](/hi/reference/configuration/#disable404route) कॉन्फिग विकल्प सेट कर सकते हैं:
+
+```js {9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: 'कस्टम 404 के साथ दस्तावेज़',
+			disable404Route: true,
+		}),
+	],
+});
+```
+
+## कस्टम फ़ॉन्ट
+
+डिफ़ॉल्ट रूप से, Starlight सभी टेक्स्ट के लिए उपयोगकर्ता के स्थानीय उपकरण पर उपलब्ध sans-serif फ़ॉन्ट का उपयोग करता है।
+यह सुनिश्चित करता है कि बड़े फ़ॉन्ट फ़ाइलों को डाउनलोड करने के लिए अतिरिक्त बैंडविड्थ की आवश्यकता के बिना, दस्तावेज़ प्रत्येक उपयोगकर्ता के परिचित फ़ॉन्ट में तेज़ी से लोड हो।
+
+यदि आपको अपनी Starlight साइट पर एक कस्टम फ़ॉन्ट जोड़ना है, तो आप कस्टम CSS फ़ाइलों में या किसी अन्य [Astro स्टाइलिंग तकनीक](https://docs.astro.build/hi/guides/styling/) के साथ उपयोग करने के लिए फ़ॉन्ट सेट कर सकते हैं। .
+
+### फ़ॉन्ट सेट करें
+
+यदि आपके पास पहले से ही फ़ॉन्ट फ़ाइलें हैं, तो [स्थानीय सेट-अप मार्गदर्शिका](#स्थानीय-फ़ॉन्ट-फ़ाइलें-सेट-करें) का पालन करें।
+Google Fonts का उपयोग करने के लिए, [Fontsource सेट-अप मार्गदर्शिका](#एक-fontsource-फ़ॉन्ट-सेट-करें) का पालन करें।
+
+#### स्थानीय फ़ॉन्ट फ़ाइलें सेट करें
+
+1. अपनी फ़ॉन्ट फ़ाइलें `src/fonts/` निर्देशिका में जोड़ें और एक खाली `font-face.css` फ़ाइल बनाएं:
+
+   <FileTree>
+
+   - src/
+     - content/
+     - fonts/
+       - **CustomFont.woff2**
+       - **font-face.css**
+   - astro.config.mjs
+
+   </FileTree>
+
+2. `src/fonts/ में अपने प्रत्येक फ़ॉन्ट के लिए एक [`@font-face` घोषणा](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) जोड़ें फ़ॉन्ट-face.css`।
```

Comment:
Looks like some formatting got mixed up here, the `src/fonts/font-face.css` path is incomplete and I think the inline code is missing its termination mark.